text/083-boost-stdlib-dev/text.md

```diff
@@ -0,0 +1,786 @@
+    theories/micromega/DeclConstantZ.v,
+    theories/micromega/OrderedRing.v, theories/micromega/Tauto.v,
+    theories/micromega/Env.v, theories/micromega/Refl.v,
+    theories/micromega/ZArith_hints.v,
+    theories/micromega/ZMicromega.v, theories/micromega/EnvRing.v,
+    theories/micromega/Lia.v, theories/micromega/VarMap.v,
+    theories/micromega/Ztac.v, theories/micromega/ZCoeff.v,
+    theories/micromega/RingMicromega.v</detail>
+* coq-roots-zarith
+  * The Coq standard library, binary integers
+  * Binary encoding of integers. This binary encoding was initially
+    developped to enable efficient computations, compared to the unary
+    encoding of `nat`. Proofs were then added making the types usable
+    for mathematical proofs, although this was not the initial
+    intent. If even-more efficient computations are needed, look at
+    the coq-roots-primitive-int package for 63 bits machine arithmetic or the
+    coq-bignums package for arbitrary precision machine
+    arithmetic. Everything can be imported with `From Coq Require
+    Import NArith` or `From Coq Require Import ZArith`. Also contains
+    the migromega tactic that can be loaded with `From Coq Require
+    Import Lia.`. Most of namespaces `Coq.NArith` and `Coq.ZArith`.
+  * 23 files, 6.5 kloc
+  * <details><summary>contains:</summary>
+    theories/Numbers/Natural/Binary/NBinary.v,
+    theories/ZArith/Zpower.v, theories/ZArith/Zquot.v,
+    theories/ZArith/Zpow_facts.v, theories/ZArith/Zgcd_alt.v,
+    theories/ZArith/Zwf.v, theories/ZArith/ZArith.v,
+    theories/ZArith/Zbitwise.v,
+    theories/Numbers/Integer/Binary/ZBinary.v,
+    theories/Numbers/Integer/NatPairs/ZNatPairs.v,
+    theories/Numbers/DecimalN.v, theories/Numbers/DecimalNat.v,
+    theories/Numbers/DecimalZ.v, theories/Numbers/DecimalFacts.v,
+    theories/Numbers/DecimalPos.v, theories/Numbers/HexadecimalN.v,
+    theories/Numbers/HexadecimalNat.v,
+    theories/Numbers/HexadecimalZ.v,
+    theories/Numbers/HexadecimalFacts.v,
+    theories/Numbers/HexadecimalPos.v, theories/btauto/Algebra.v,
+    theories/btauto/Reflect.v, theories/btauto/Btauto.v</details>
+* coq-roots-qarith-base
+  * The Coq standard library, binary rational numbers base package.
+    Use coq-roots-qarith for more functionalities with the `field`
+    and `Lqa` tactics. Namespace `Coq.QArith`.
+  * 4 files, 1.8 kloc
+  * <details><summary>contains:</summary>theories/QArith/Qreduction.v,
+    theories/QArith/Qminmax.v, theories/QArith/QOrderedType.v,
+    theories/QArith/QArith_base.v</details>
+* coq-roots-field
+  * The Coq standard library, `field` tactic.
+  * 9 files, 3.8 kloc
+  * <details><summary>contains:</summary> theories/QArith/Qfield.v,
+    theories/QArith/Qround.v, theories/QArith/Qring.v,
+    theories/QArith/Qpower.v, theories/QArith/Qcanon.v,
+    theories/setoid_ring/Field_theory.v,
+    theories/setoid_ring/Field_tac.v, theories/setoid_ring/Rings_Q.v,
+    theories/setoid_ring/Field.v</details>
```

Comment:
It is unfortunate that field comes bundled with Q. Needn't do anything about it now, just another case where package the package boundaries are not really good for external consumption even though they would be immediately useful internally.